workflow_scripts/test_models.py

```diff
@@ -55,25 +81,40 @@ def main():
                 if (args.target == 'onnxruntime' or args.target == 'all'):
                     # finally check the ONNX model from .tar.gz by ORT
                     # if the test_data_set does not exist, create the test_data_set
-                    check_model.run_backend_ort(model_path_from_tar, test_data_set)
-                    print('[PASS] {} is checked by onnxruntime. '.format(model_name))
+                    try:
+                        check_model.run_backend_ort(model_path_from_tar, test_data_set)
+                        print('[PASS] {} is checked by onnxruntime. '.format(model_name))
+                    except Exception as e:
+                        if not args.create:
+                            raise Exception(e)
```

Comment:
No need to wrap the original exception if you're not adding any info or changing the type:
```suggestion
                            raise
```